changelog.d/1310.change.md

```diff
@@ -0,0 +1 @@
+Speed up the generated `__eq__` methods significantly.
```

Comment:
We should document the change in behavior here. You can make it a breaking fragment, I don’t care that much, but it should definitely be there.

there might be some docs that claim the tuple way too?